ui/src/components/searchbox/QasSearchBox.stories.js

```diff
@@ -0,0 +1,76 @@
+import QasSearchBox from './QasSearchBox.vue'
+
+export default {
+  title: 'Components/SearchBox',
+  component: QasSearchBox,
+  parameters: {
+    docs: { 
+      description: { 
+        component: 'Research Component'
+                    + '<br> Dependencies: https://fusejs.io/' 
+      } 
+    }
+  },
+  
+  argTypes: {
+    list: {
+      name: 'list',
+      description: 'Array of objects containing label and value'
+    },
+    fuseOptions: {
+      name: 'fuseOptions',
+      description: 'Indicates which parameter to use for the filter https://fusejs.io/api/options.html',
+    },
+    placeholder: {
+      description: 'Standard input text'
+    },
+    height: {
+      description: 'Component height'
+    },
+    value: {
+      description: 'Base value for research'
+    },
+    empty:{
+      table:{
+        disable: true
+      }
+    },
+    default:{
+      table:{
+        disable: true
+      }
+    },
+    input:{
+      table:{
+        disable: true
+      }
+    },
+    emptyResult:{
+      table:{
+        disable: true
+      }
+    }
+  }
+}
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { QasSearchBox },
+  template:
+   `<QasSearchBox v-bind="$props" > 
+      <template v-slot="{ results }">
+        <div v-for="(obj, index) in results" :key="obj.index">
```

Comment:
Por que abreviar `obj`?

Reply:
v-for removido, era um teste, vai retornar somente o results, usar o v-for aqui não faz muito sentido pois não faz parte do componente a listagem.